determined-ai/determined
# fix: edit some libs in package.json

## Description
I found some libraries in devDependencies which should be in Dependencies. I moved some libraries and removed an unused library  



## Test Plan
- Check if these moves are valid 
- Check if there are any libraries I should move (or not move)
- Check if library update won't break anything
  - throttle-debounce (3.0.1 -> 5.0.0): https://github.com/niksy/throttle-debounce/releases
  - path-to-regexp (6.2.0 -> 6.2.1): https://github.com/pillarjs/path-to-regexp/releases
- Check if library removal won't break anything
  - Sinon: I removed it because this lib is not used anywhere



## Commentary (optional)




## Checklist

See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.





[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
